src/main.py

```diff
@@ -2,8 +2,8 @@
 import src.image_operations as image_operations
 import src.pca as pca
 import src.load_image_vectors as load_image_vectors
-import src.pickle_operations as pickle_io
 
+import src.pickle_operations as pickle_io
 tests_count = 0
 tests_success = 0
```

Comment:
not needed anymore

Reply:
why? the main still reads the images from pickle files